apps/renderscript/README.md

```diff
@@ -0,0 +1,40 @@
+Halide Renderscript "blur" and "copy" samples
+===
+
+These samples demonstrates two supported modes of operation for Renderscript codegen:
+ - vectorized operations over interleaved RGBA images;
+ - single-element operations over planar images.
+
+Prerequisites
+---
+
+You will need to have Android NDK in the path.
+This app at the moment only runs on arm-based Android devices.
+TODO(aam): Investigate and confirm that the app runs on x86 Android devices as well.
+
+Building and running
+---
+
+```
+$ make
+...
+Planar blur:
```

Comment:
Well, I wanted to show how to run the test app and what to look for in the output.